kombu/transport/redis.py

```diff
@@ -1429,7 +1429,7 @@ def _sentinel_managed_pool(self, asynchronous=False):
 
         return sentinel_inst.master_for(
             master_name,
-            self.Client,
+            redis.StrictRedis,
```

Comment:
should we also adjust some test cases to verify this change?

Reply:
Added tests...